sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs

```diff
@@ -448,7 +448,7 @@ private async Task ApplyPlugins(IReadOnlyList<ServiceBusReceivedMessage> message
                 DiagnosticProperty.ProducerKind);
             scope.Start();
 
-            IReadOnlyList<ServiceBusReceivedMessage> messages = new List<ServiceBusReceivedMessage>();
+            IReadOnlyList<ServiceBusReceivedMessage> messages;
```

Comment:
I assume this is okay because InnerReceiver.PeekMessagesAsync will never return null?